File: FateGrandAutomata/DraggableRecyclerView/ItemViewHolder.cs
using Android.Graphics;
using Android.Views;
using Android.Widget;
using AndroidX.RecyclerView.Widget;

namespace FateGrandAutomata
{
    public class ItemViewHolder : RecyclerView.ViewHolder, IItemTouchHelperViewHolder
    {
        public TextView TextView { get; }

        public ImageView ImageView { get; }

        public ItemViewHolder(View ItemView) : base(ItemView)
        {
            TextView = ItemView.FindViewById<TextView>(Resource.Id.card_priority_textview);
            ImageView = ItemView.FindViewById<ImageView>(Resource.Id.card_priority_dragger);
        }

        public void OnItemSelected()
        {
            ItemView.SetBackgroundColor(Color.LightGray);
        }

        public void OnItemClear()
        {
            ItemView.SetBackgroundColor(Color.Transparent);
        }
    }
}

File: app/src/main/java/com/mathewsachin/fategrandautomata/ui/card_priority/ItemViewHolder.kt
package com.mathewsachin.fategrandautomata.ui.card_priority

import android.graphics.Color
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.mathewsachin.fategrandautomata.R
import com.mathewsachin.fategrandautomata.util.IItemTouchHelperViewHolder

class ItemViewHolder(ItemView: View) : RecyclerView.ViewHolder(ItemView), IItemTouchHelperViewHolder {
    val textView: TextView = ItemView.findViewById(R.id.card_priority_textview)
    val imageView: ImageView = ItemView.findViewById(R.id.card_priority_dragger)

    override fun onItemSelected() {
        itemView.setBackgroundColor(Color.LTGRAY)
    }

    override fun onItemClear() {
        itemView.setBackgroundColor(Color.TRANSPARENT)
    }
}